#!/usr/bin/env python

PACKAGE = "last_letter_2"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("roll_kp",  double_t, 0, "kp parameter of roll PD", 	.12,  0, 2)
gen.add("roll_kd",  double_t, 0, "kd parameter of roll PD", 	.04,  0, 2)
gen.add("pitch_kp",  double_t, 0, "kp parameter of pitch PD", 	.12,  0, 2)
gen.add("pitch_kd",  double_t, 0, "kd parameter of pitch PD", 	.04,  0, 2)
gen.add("yaw_kp",  double_t, 0, "kp parameter of yaw PD", 	.14,  0, 2)
gen.add("yaw_kd",  double_t, 0, "kd parameter of yaw PD", 	.07,  0, 2)
gen.add("alt_kp",  double_t, 0, "kp parameter of altitude hold PD", 	.3,  0, 2)
gen.add("alt_kd",  double_t, 0, "kd parameter of altitude hold PD",	.25,  0, 2)

exit(gen.generate(PACKAGE, "quadcopter_contr_node", "PD_gains"))